pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
-name = "bg-atlasapi"
-description = "A lightweight python module to interact with atlases for systems neuroscience"
+name = "brainglobe-atlasapi"
+description = "A lightweight python module to interact with atlases for systems neuroscience, and for generating atlases for the API."
```

```suggestion
description = "A lightweight python module to interact with and generate atlases for systems neuroscience."
```
Shorter and less repetition within the sentence? (Just a suggestion)